airbyte-tests/src/acceptanceTests/java/io/airbyte/test/acceptance/AcceptanceTests.java

```diff
@@ -467,7 +467,8 @@ public void testManualSync() throws Exception {
     catalog.getStreams().forEach(s -> s.getConfig().syncMode(syncMode).destinationSyncMode(destinationSyncMode));
     final UUID connectionId =
         createConnection(connectionName, sourceId, destinationId, List.of(operationId), catalog, null).getConnectionId();
-
+    // Avoid Race condition with the new scheduler
+    Thread.sleep(10 * 1000);
```

Comment:
weird. what exactly is the issue? the connection is created but not in temporal yet? should `createConnect` just not return until everything is set up properly?

Reply:
The connection is created, which means that the a workflow is plan. The temporal workflow runs asynchronously, that's why I put this wait to make sure that the job gots created when we call `final JobInfoRead connectionSyncRead = apiClient.getConnectionApi().syncConnection(new ConnectionIdRequestBody().connectionId(connectionId));` because if it was done without a wait the async temporal process may have done it or not. CreateConnection might return the UUID of a connection for which the job is not created yet